Application/Services/ActionNotesService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using CarSharingApp.Application.Interfaces;
+using CarSharingApp.Domain.Abstractions;
+using CarSharingApp.Domain.Entities;
+
+namespace CarSharingApp.Application.Services
+{
+    public class ActionNotesService : IActionNotesService
+    {
+        private readonly IRepository<ActionNote> _noteRepository;
+
+        public ActionNotesService(IRepository<ActionNote> noteRepository)
+        {
+            _noteRepository = noteRepository;
+        }
+
+        public async Task<List<ActionNote>> GetCustomerNotesWithLimitConnectedWithAccount(Guid customerId, int skip, int limit)
+        {
+            var result = await _noteRepository.GetAllAsync(n => (n.ActorId == customerId || n.ActionEntityId == customerId)
+                && 10 <= n.Type && n.Type <= 19);
```

Comment:
Document how to use these magic numbers such as 10, 19
define either constants or fields